apps/interactor/docs/docker.rst

```diff
@@ -17,8 +21,8 @@ The following may be used::
         --restart=always \
         --net=host \
         -e TZ=Australia/Melbourne \
-        -v $PWD/configdir:/project/config \
-        delfick/lifx-photons-interactor:0.8.4
+        -v "${PWD}/configdir":/project/config \
+        delfick/lifx-photons-interactor
```

Comment:
I keep putting in the exact version so that when I update this points to the latest. I'm a big fan of pinning exact versions of things than relying on a moving target.

Reply:
I have restored the version and have suppressed the urge to get on my soapbox about containers and versions and anti-patterns. You're welcome.

Reply:
thanks :)